cmd/server/grpc/main.go

```diff
@@ -351,3 +365,56 @@ func (s *server) Signal(stream pb.SFU_SignalServer) error {
 		}
 	}
 }
+
+// Relay endpoint implementation
+func (s *Server) Relay(ctx context.Context, request *pb.RelayRequest) (*pb.RelayResponse, error) {
+
+	conn, err := grpc.Dial(request.Sfu, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "%s", err)
+	}
+	defer conn.Close()
+
+	sfuClient := pb.NewSFUClient(conn)
```

Comment:
lets cache this in a map `server`. we can use `request.Sfu` as key. if it already exists, we dont need to create a new client

Reply:
okay.